src/main/java/perkpack/controllers/CardController.java

```diff
@@ -0,0 +1,60 @@
+package perkpack.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import perkpack.models.Card;
+import perkpack.repositories.CardRepository;
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("cards")
+public class CardController {
+
+    private final CardRepository cardRepository;
+
+    @Autowired
+    public CardController(CardRepository cardRepository){
+        this.cardRepository = cardRepository;
+    }
+
+
+    @PostMapping
+    public Card createCard(@Valid @RequestBody Card card){
```

Comment:
We should probably include a check that prevents creating a card if one already exists with the same name.

Reply:
Check isn't needed until we take user input on card creation.